amazonec2/Module.md

```diff
@@ -49,231 +49,217 @@ in the Amazon EC2 client config.
+The `createImage` remote function will create an image. It returns a `Image` object with created image id if it is successful or an `error` if unsuccessful.
 
 ```ballerina
+var newImage = amazonEC2Client->createImage("ami-0ba4ce8cbsffd4e6333", "Test Image");
 if (newImage is amazonec2:Image) {
-    io:println(" Successfully create a new image : ");
-    id = newImage.imageId;
-    io:println(newImage);
+    io:println("Successfully created a new image: ", newImage);
 } else {
-    test:assertFail(msg = <string>newImage.detail().message);
+    io:println("Error: ", newImage.detail().message);
 }
 ```
 
-The `describeImages` function will describe the images . It returns an `Image` array
-with image details if it is successful or the response is a `error`.
+The `describeImages` remote function will describe the images. It returns an `Image` array with image details if it is successful or an `error` if unsuccessful.
 
 ```ballerina
-if (describeInstances is error) {
-    test:assertFail(msg = <string>describeInstances.detail().message);
+var describeImageResponse = amazonEC2Client->describeImages("ami-0ba4ce8cb48ssfsfs");
+if (describeImageResponse is amazonec2:Image[]) {
+    io:println("Successfully described the image: ", describeImageResponse);
 } else {
-     io:println("Successfully describe the instance : ");
-     io:println(describeInstances);
+    io:println("Error: ", describeImageResponse.detail().message);
 }
 ```
 
-The `describeImageAttribute` function will describe an image with specified attributes . It returns an `ImageAttribute` response based on the attribute name if it is successful or the response is a `error`.
+The `describeImageAttribute` remote function will describe an image with specified attributes. It returns an `ImageAttribute` object based on the attribute name if it is successful or an `error` if unsuccessful.
 
 ```ballerina
+var imageAttributeResponse = amazonEC2Client->describeImageAttribute("ami-0ba4ce8cb48ssfsfs", "Description of Image");
 if (imageAttributeResponse is amazonec2:ImageAttribute) {
-    io:println(" Successfully describes an image with an attribute : ");
-    io:println(imageAttributeResponse);
+    io:println("Successfully described an image with an attribute: ", imageAttributeResponse);
 } else {
-    test:assertFail(msg = <string>imageAttributeResponse.detail().message);
+    io:println("Error: ", imageAttributeResponse.detail().message);
 }
 ```
 
-The `deRegisterImage` function will deregisters the specified AMI.
-After you deregister an AMI, it can't be used to launch new instances;
-however,it doesn't affect any instances that you've already launched from the AMI.
-It returns true as a service response if it is successful or the response is a `error`.
+The `deRegisterImage` remote function will de registers the specified AMI. After you deregister an AMI, it can't be used to launch new instances. However, it doesn't affect any instances that you've already launched from the AMI.
```

Comment:
```suggestion
The `deRegisterImage` remote function will deregister the specified AMI. After you deregister an AMI, it cannot be used to launch new instances. However, it doesn't affect any instances that you've already launched from the AMI.
```

We might also have to rename this  function as `deregisterImage`. Can you please create an issue to track this?

Reply:
Done.